src/bloqade/pyqrack/task.py

```diff
@@ -56,7 +56,7 @@ def qubits(self) -> list[PyQrackQubit]:
             Warning("Task has not been run, there are no qubits!")
             return []
 
-    def batch_run(self, shots: int = 1) -> dict[RetType, float]:
+    def batch_run(self, shots: int = 1, **kwargs) -> dict[RetType, float]:
```

Comment:
```suggestion
    def batch_run(self, shots: int = 1, **kwargs) -> dict[RetType, float]:
```

we should add type hints to `**kwargs` this is a must do before merging.

Reply:
Oops, I just realized that `run()` takes no arguments, so `batch_run` does not need to pass those in either.